code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -214,6 +214,8 @@ var/global/list/ts_spiderling_list = list()
 				var/mob/living/carbon/human/H = G
 				if(!(H.dna.species.reagent_tag & PROCESS_ORG) || (!H.dna.species.tox_mod))
 					can_poison = 0
+				if(H.stat != DEAD && H.health < HEALTH_THRESHOLD_DEAD) // Intentionally not HEALTH_THRESHOLD_CRIT, as that is softcrit
```

Comment:
may want to make this 

`if(!H.check_death_method() && H.health <= HEALTH_THRESHOLD_DEAD) H.death()`

Up to you, of course, but the above code basically will only call `death` on mobs that utilize the new crit method; mobs that use the old method would be immune to this.

I'd still make it `<=`, regardless of what you choose though.

Reply:
Changed.